src/App.jsx

```diff
@@ -4,6 +4,7 @@ import { Navbar } from "./components/Navbar";
 import { Home } from "./components/home";
 import { useState } from "react";
 import { LoginSignup } from "./components/LoginSignup";
+import { SingleArticle } from "./components/SingleArticle";
```

Comment:
You have a couple incorrect imports due to using lowercase first letters in the paths for home and articles. Please change these as they are stopping your app from working